spyder/widgets/editor.py

```diff
@@ -359,9 +360,21 @@ def __init__(self, parent, actions):
         close_all_but_this = create_action(self, _("Close all but this"),
                                            triggered=self.close_all_but_this)
 
-        self.menu_actions = actions + [None, fileswitcher_action,
+        actions = actions + [None, fileswitcher_action,
                                        symbolfinder_action,
-                                       copy_to_cb_action, None, close_right,
+                                       copy_to_cb_action]
+        
+        if sys.platform == 'darwin':
+            show_in_finder_action = create_action(self, _("Show in finder"), 
+                                              triggered=self.showInFinder)
+        else:
+            show_in_finder_action = create_action(self, 
+                                          _("Show in external file explorer"), 
+                                              triggered=self.showInFinder)

```

Comment:
You don't need to create two actions. Only one action is needed and you change its text according to the operating system, i.e. something like:

```python
if sys.platform == 'darwin':
    text = _("Show in finder")
else:
    text = _("Show in external file explorer")
external_fileexp_action = create_action(self, text,
                            triggered=self.show_in_external_file_explorer)
```